documentation/source/triggers.rst

```diff
@@ -36,11 +36,11 @@ Timing
 
 .. autoclass:: cocotb.triggers.Timer
 
-.. autoclass:: cocotb.triggers.ReadOnly
+.. autoclass:: cocotb.triggers.ReadOnly()
 
-.. autoclass:: cocotb.triggers.ReadWrite
+.. autoclass:: cocotb.triggers.ReadWrite()
 
-.. autoclass:: cocotb.triggers.NextTimeStep
+.. autoclass:: cocotb.triggers.NextTimeStep()
```

Comment:
I don't like that Sphinx leaves off the parameter list parens if there are no parameters, but `help()` does that as well, so I guess it is the way it is.

Reply:
Me neither: I'd encourage you to file a patch against sphinx to fix it.